transport/wseb/src/test/scripts/org/kaazing/gateway/transport/wseb/echo.text.and.binary.rpt

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2007-2014, Kaazing Corporation. All rights reserved.
+#
+
+# create request for text
+
+connect tcp://localhost:8000
+connected
+
+write "GET /echo/;e/ct HTTP/1.1\r\n"
+write "Accept: */*\r\n"
+write "Accept-Encoding: gzip,deflate,sdch\r\n"
+write "Accept-Language: en-US,en;q=0.8\r\n"
+write "Connection: keep-alive\r\n"
+write "Host: localhost:8000\r\n"
+write "Referer: http://localhost:8000/?.kr=xs\r\n"
+write "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/29.0.1547.65 Safari/537.36\r\n"
+write "X-Origin: http://localhost:8000\r\n"
+write "X-WebSocket-Version: wseb-1.0\r\n"
+write "\r\n"
+
+read "HTTP/1.1 200 OK\r\n"
+read "Cache-Control: no-cache\r\n"
+read "Content-Length: 196\r\n"
+read "Content-Type: text/plain;charset=UTF-8\r\n"
+read /Date: .*/ "\r\n"
```

Comment:
as per Chris Barrow's advice, this should be changed to pull the \r\n into the expression